packages/reference/src/common/MultipleReferenceEditor.tsx

```diff
@@ -24,6 +24,21 @@ type EditorProps = ReferenceEditorProps &
     children: (props: ReferenceEditorProps & ChildProps) => React.ReactElement;
   };
 
+function onLinkOrCreate(
+  setValue: ChildProps['setValue'],
+  entityType: ChildProps['entityType'],
+  items: ChildProps['items'],
+  ids: string[],
+  index = -1
```

Comment:
Why not `index?: number` instead? Might be also confusing as I saw it being used as `insert before the first from end`

Reply:
Sure - I'm happy with rewriting. My thought was to echo the behavior of primitives like `indexOf`, but I can see why this would be confusing or conflict with the ability to do Ruby-like backward index scanning at some point in the future.

Reply:
Fixed in https://github.com/contentful/field-editors/pull/442/commits/c166bdc923aef114f88c9fdeb0b63a2bd25302e7.

Reply:
Yeah, I agree with using `-1` as a search result, but for inserting imho it becomes a different meaning.